app/models/menu.rb

```diff
@@ -0,0 +1,5 @@
+class Menu < ApplicationRecord
+  belongs_to :restaurant
+
+  validates_presence_of :name, presence: true
```

Comment:
😜 validation of presence implies that it needs to be, uhmmm... you know... present. Right. You still think you need to set it to 'true'. Are you sure it's necessary?